### PR TITLE
[Robot] Verify new Education Cloud Settings

### PR DESCRIPTION
**Pull request by rhakeshj**
Added tests for below sections in new EDA settings

- Addresses
- Course Connections
- Errors
- EDC Home
- Program Plans
- Relationships
- System Tools

Removed the old eda settings tests and fixed navigation to address 2gp and non 2gp orgs for new settings

# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Review by Safiyab — APPROVED** [review]

File: robot/EDA/resources/EDA.py [hunk 1]
```diff
@@ -427,7 +437,7 @@ def click_on_hub_link(self, field, link):
     def go_to_settings_health_check(self):
         """ Navigates to the Home view of Settings Health Check app"""
         url = self.cumulusci.org.lightning_base_url
-        url = "{}/lightning/n/Settings_Health_Check".format(url)
+        url = "{}/lightning/cmp/c__HealthCheckContainer".format(url)
```
[the remark this answers is not in the document]
Comment (rhakeshj) [reply robot/EDA/resources/EDA.py]: Thanks for catching it! Updated similar to EDA settings app.

File: robot/EDA/resources/EDA.py [hunk 2]
```diff
@@ -662,3 +675,28 @@ def update_settings_dropdown_value(self,**kwargs):
...
+            the button from test.
+        """
+        for field,button in kwargs.items():
+            locator = eda_lex_locators["eda_settings_new"]["update_button"].format(field,button)
+            self.selenium.wait_until_page_contains_element(locator, timeout=60)
+            self.selenium.wait_until_element_is_visible(locator,
+                                                error=f"'{field}' field is not available ")
+            self.selenium.click_element(locator)
+            self.click_footer_button(value)
+            self.verify_custom_toast_message_displayed
```
[the remark this answers is not in the document]
Comment (rhakeshj) [reply robot/EDA/resources/EDA.py]: Done

File: robot/EDA/resources/EducationCloudSettingsPageObject.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
+from BaseObjects import BaseEDAPage
+from EDA import eda_lex_locators
+from cumulusci.robotframework.pageobjects import HomePage
+from cumulusci.robotframework.pageobjects import pageobject
+import time
+from datetime import date
+
+
+@pageobject("Home", "Education Cloud Settings")
```
[the remark this answers is not in the document]
Comment (rhakeshj) [reply robot/EDA/resources/EducationCloudSettingsPageObject.py:9]: Yeah. Education Cloud Settings is like a home/landing page from where a user can launch EDA's settings page, documentation, Settings Health Check app etc.

File: robot/EDA/resources/EducationCloudSettingsPageObject.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            if field == "Resources":
+                locator = eda_lex_locators["eda_settings_new"]["resources_tile"].format(field,value)
+                self.selenium.wait_until_page_contains_element(locator, timeout=60, error=f'{locator} is not available')
+                self.selenium.wait_until_element_is_visible(locator,
+                                                error= "Element is not displayed for the user")
+            else:
+                locator = eda_lex_locators["eda_settings_new"]["other_tile"].format(field,value)
+                self.selenium.wait_until_page_contains_element(locator, timeout=60, error=f'{locator} is not available')
+                self.selenium.wait_until_element_is_visible(locator,
+                                                error= "Element is not displayed for the user")
```
[the remark this answers is not in the document]
Comment (rhakeshj) [reply robot/EDA/resources/EducationCloudSettingsPageObject.py]: I was trying to do something similar (using union, or  operator) but couldn't get a working xpath. Good to know and thanks for sharing it.

File: robot/EDA/tests/browser/settings/relationships/reciprocal_prevent_duplicate.robot [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+Suite Setup     Open Test Browser
+Suite Teardown  Run Keywords
+...             Update settings to out of the box values        AND
+...             Capture screenshot and delete records and close browser
+
+Test Setup      Go to education cloud settings
+
+*** Keywords ***
+Update settings to out of the box values
+    Go to education cloud settings
```
[the remark this answers is not in the document]
Comment (rhakeshj) [reply robot/EDA/tests/browser/settings/relationships/reciprocal_prevent_duplicate.robot]: Done

**Review by spinkelman — APPROVED** [review]
After Functional Review - this is approved.

**Pull request merged**